dataclients/kubernetes/kube_test.go

```diff
@@ -855,10 +855,10 @@ func TestHealthcheckInitial(t *testing.T) {
 
 		r, err := dc.LoadAll()
 		if err != nil {
```

Comment:
better delete "use healthcheck, fail" and "use healthcheck, reload fail" tests. See the explanation in the other comment.